squid/Dockerfile

```diff
@@ -7,9 +7,13 @@ FROM node-with-gyp AS builder
 WORKDIR /squid
 ADD package.json .
 ADD package-lock.json .
-RUN npm ci
 ADD tsconfig.json .
 ADD src src
+ADD typegen.json .
```

Comment:
I wonder if this is needed or not. Types are only needed for development, once the JS is generated (with build) no need for this)

Reply:
That stage of the Dockerfile called `builder` is the responsible to build the application and generate the distributable files that will be deployed to the servers.

I'd say, if `typegen.json` is necessary to build the app, let's leave it as it, but, if you say that this file is not mandatory to build the app, I can remove it.

Two lines below, I am running:
```dockerfile
RUN npm run codegen
RUN npm run typegen
```

Let me know if I should remove the typegen command there also.